plugins/generator-1.21.1/neoforge-1.21.1/procedures/utils/world/entity_in_range.java.ftl

```diff
@@ -0,0 +1,4 @@
+private static Entity findEntityInWorldRange(LevelAccessor world, Class<? extends Entity> clazz, double x, double y, double z, double range) {
+	return (Entity) world.getEntitiesOfClass(clazz, AABB.ofSize(new Vec3(x, y, z), range, range, range), e -> true)

```

Comment:
This method is single line, so it should not need this method and can be inlined and part of #5253

Reply:
We have to use either coordinates or vector twice, separate method allows to only parse their values once. These are also referenced in lambda below, so making them method arguments ensures they are final when lambda uses them.

Reply:
Makes sense